src/tensor.c

```diff
@@ -851,6 +851,8 @@ uint ParseTensorGetArgs(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
         RedisModule_WrongArity(ctx);
         return fmt;
     }
+    if (argc == 2)
+        fmt = TENSOR_BLOB | TENSOR_META;
```

Comment:
```suggestion
    if (argc == 2) return TENSOR_BLOB | TENSOR_META;
```
also, one-liner or open scope after line drop